src/OpenDB/test/gcd_abstract_lef.lefok

```diff
@@ -0,0 +1,812 @@
+VERSION 5.8 ;
+BUSBITCHARS "[]" ;
+DIVIDERCHAR "/" ;
+MACRO gcd
+  FOREIGN gcd 0 0 ;
+  CLASS BLOCK ;
+  ORIGIN 0 0 ;
+  SIZE 200.26 201.6 ;
+  PIN VDD
+    USE POWER ;
+    DIRECTION INOUT ;
+    PORT
+      VIA 136.14 106.23 via6_960x2800 ;
```

Comment:
Generally LEF abstracts don't include via blockages because it ends up being redundant information - if the layer above and the layer below is blocked, then one can't insert a via anyways. Might be good to fix in a future PR.